comps/comps-foreman-rhel6.xml

```diff
@@ -170,6 +173,8 @@
       <packagereq type="default">rubygem-awesome_print</packagereq>
       <packagereq type="default">rubygem-bundler_ext</packagereq>
       <packagereq type="default">rubygem-clamp</packagereq>
+      <packagereq type="default">rubygem-concurrent-ruby</packagereq>
```

Comment:
Ah, this is a little tricky.. the non-SCL version is still really only needed in plugins.  I think it'll have to stay here (in core), so can you remove it from plugins too please?